smart-contracts/developing-contracts/verify-a-contract.md

```diff
@@ -25,6 +25,196 @@ The following guide walks you through the process of contract verification using
 * [Remix](https://remix.ethereum.org/)
 
 ### Procedure
+#### Via. API (Recommended)
+
+This section provides detailed documentation for the Filfox Contract Verification API. The API allows you to verify your smart contracts programmatically, and is the recommended method for verifying contracts. Below, you will find information on how to use the API, including the available endpoints, request and response formats, and example code snippets.
+
+Please refer to the [documentation](https://filfox.info/api/v1/docs/static/index.html) for more information on how to verify contracts using the Filfox API.
```

Comment:
This documentation links to Filfox API Documents which does not include verifyContract API. Is it a wrong link? Or maybe we can just remove it.

Reply:
@xBalbinus can you address the comments, so we can get this merged?